index.html

```diff
@@ -937,6 +937,11 @@ <h2>DID Parameters</h2>
 did:example:123?versionTime=2021-05-10T17:00:00Z
         </pre>
 
+        <pre class="example nohighlight"
+          title="A DID URL with a 'service' and a 'relativeRef' DID parameter">
+did:example:123?service=agent&relativeRef=/credentials
```

Comment:
```suggestion
did:example:123?service=agent&relativeRef=/credentials
```

This was removed because we lack a specification or an entry in the DID Specification Registries that shows how this would work. I'm a -1 to this until the WG decides that we are going to allow examples that are not tied to things in the DID Spec Registries, or we have something in DID Spec Registries to point to wrt. the example.

Reply:
https://w3c.github.io/did-spec-registries/#relative-ref-param

I think the only issue is the kebab to camel case change.

Reply:
```suggestion
did:example:123?service=website&relativeRef=/credentials
```

Can we do something like the above?

Reply:
I am in favor of this change, or "service-provider" agent has the benefit of not implying http